src/sensai/hyperopt.py

```diff
@@ -152,17 +162,15 @@ def _saveCSV(self):
                 os.makedirs(dirname, exist_ok=True)
             self.df.to_csv(self.csvPath, index=False)
 
-    def addModelParamsMetrics(self, model, params, metrics):
-        values = dict(metrics)
-        values.update(params)
-        values["str(model)"] = str(model)
-        self.addValues(values)
-
     def getDataFrame(self) -> pd.DataFrame:
         return self.df
 
 
 class GridSearch(TrackedExperimentDataProvider):
+    """
+    Instances of this class can be used for evaluating clustering models with different user-provided parametrizations
```

Comment:
This class is for all types of models not only clustering models.

Reply:
Whoops, classical copy paste error